### PR TITLE
Jotai: Docs for debugLabel Babel plugin

### PR DESCRIPTION
**Pull request by Thisen**
Add docs for `jotai/babel/plugin-debug-label`.

### CODE REVIEW

**Comment by Thisen** [comment]
Please review @dai-shi.

File: docs/jotai/api/babel.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: Babel
+description: This doc describes the `jotai/babel` bundle.
+nav: 2.4
+---
```
Comment (CodyJasonBennett) [inline docs/jotai/api/babel.mdx:5]: Doesn't look like formatting is well supported in the current description component.

image

I'll see if I can run a fix by.

Comment (CodyJasonBennett) [reply docs/jotai/api/babel.mdx:5]: Fixed this in db71847346ee95836b230afac89633f1676d1532. Carry on 😅.

File: docs/jotai/api/babel.mdx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## plugin-debug-label
+
+Jotai is based on object references and not keys (like Recoil). This means there's no identifier for atoms. To identify atoms, it's possible to add a `debugLabel` to an atom, which can be found in React devtools.
```
Comment (dai-shi) [inline docs/jotai/api/babel.mdx:9]: I'm not sure if "identify" is the right word. debugLabels don't need to be unique unlike keys.

**Review by dai-shi — APPROVED** [review]
Looks good for the initial version. Let's keep improving it.

**Pull request merged**